examples/seq2seq/run_seq2seq.py

```diff
@@ -561,13 +591,10 @@ def compute_metrics(eval_preds):
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
         trainer.save_model()  # Saves the tokenizer too for easy upload
 
-        output_train_file = os.path.join(training_args.output_dir, "train_results.txt")
+        metrics = train_result.metrics
+        metrics["train_n_objs"] = data_args.max_train_samples
```

Comment:
yes, I need this information when comparing results. it's too easy to git mixed up in the args being changed, so the important params should be bundled with the metrics IMHO.

i.e. it can be essential to know how much data was fed to the script to know when things go wrong. at least in my experience so far I often refer to this number when analyzing results.

Reply:
but you have an important point there - what if the total number of samples was less than max? could we report this in metrics?

Reply:
It would be `min(data_args.max_train_samples, len(train_dataset))` in that case.

Reply:
Awesome - thank you for the tip and I renamed it too so it's consistent with `samples_per_sec`